bitski-local/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "bitski-local"
 version = "0.1.0"
 edition = "2021"
+publish = false
```

Comment:
why isn't this published?

Reply:
Cargo won't publish it because it has a git dependency, so this flag is just a sanity check

Reply:
Removed b/c it's redundant

Reply:
ah yeah I forgot about the git dependency